app/views/reviews/_requires_feedback.html.erb

```diff
@@ -1,48 +1,20 @@
 <tr class="user_input">
 
-  <script>
-
-    function addFeedbackClickHandler(cellId, checkId, pId, iconId) {
-      document.getElementById(cellId).onclick = function() {
-      activateSaveButton();
-    
-        var checkbox = document.getElementById(checkId);
-        checkbox.checked = !checkbox.checked;
-        
-        var iconClass = checkbox.checked ? 'eui-icon eui-check-o' : 'fa fa-comments';
-        document.getElementById(iconId).className = iconClass;
-
-        var cellEl = document.getElementById(cellId), cellClass = "review-toggle__cell";
-        var textEl = document.getElementById(pId), textClass = "review-toggle__text";
-        
-        if (checkbox.checked) {
-          cellEl.classList.add(cellClass);
-          textEl.classList.add(textClass);
-        } else {
-          cellEl.classList.remove(cellClass);
-          textEl.classList.remove(textClass);
-        }
-      }; 
-    }
-
-  </script>
-
   <th class="headcol headcol__user-row">Feedback</th>
   
   <% section_titles.each do |title| %>
     <td class="no_padding column_<%=title%>">
-      <div id="feedback_cell_<%= title %>" class="feedback_cell review-toggle">
+      <div id="feedback_cell_<%= title %>" class="<%= feedback_cell_class(current_user) %>"
+           onclick='feedbackClickHandler("feedback_cell_<%= title %>", "feedback_check_<%= title %>", "feedback_text_<%= title %>", "feedback_icon_<%= title %>", <%= current_user.daac_curator? %>)'>
```

Comment:
My recollection is hazy, but I think we will be revisiting this line when we need to update for the security header (I think onclicks were regarded as just as bad as script tags).  I would definitely prefer to see it resolved, but I'm still thinking about whether I think it needs to be.

Reply:
I think this is ok for now, but it's still adding javascript in the div tag. I would still advocate for writing a ticket to revisit, especially if it is used in a few other places -- I'd be happy to take a stab at it.

Reply:
Thanks, I removed the JS from the erb file but I ran into an issue with registering the onclick via jquery (seemed to be registering it twice, so for every click event, the method would fire twice, tried to track it down but to no avail), so I'll revisit this issue later.  The input tag still has the onclick in there but at least there is no JS.    Please take a look at the latest implemenation.